python/ray/ray_constants.py

```diff
@@ -81,3 +82,9 @@ def env_integer(key, default):
 NO_RECONSTRUCTION = 0
 # A constant indicating that an actor should be reconstructed infinite times.
 INFINITE_RECONSTRUCTION = 2**30
+
+# This definition should be the same with gcs.fbs.
+# The other language types could be used in cross-language ray call.
+LANGUAGE_PYTHON = 0
+LANGUAGE_CPP = 1
+LANGUAGE_JAVA = 2
```

Comment:
We shoudln't need to define the constants. See how this is done in `gcs_utils.py`, see can use `ray.gcs_utils.TablePubsub.HEARTBEAT_BATCH` to access other enums defined in `gcs.fbs`. Let's do something similar for the language enum.